UI/src/view/point_asset/directionalTrafficSignForm.js

```diff
@@ -14,17 +14,19 @@
       me.selectedAsset = parameters.pointAsset.selectedPointAsset;
     };
 
-    this.renderValueElement = function(asset) {
+    //TODO: investigate DirectionalTrafficSign to be created by point asset form
+    var propertyOrdering = ['suggest_box'];
+
+    this.renderValueElement = function(asset, collection, authorizationPolicy) {
```

Comment:
The unused function warning is appearing because there is no direct call for this method in the current asset. However this will be call by it's parent in PointAssetForm